target/linux/rockchip/base-files/etc/board.d/02_network

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
If you use a subtarget, please move all base-files to the subtarget directory, so they are not automatically used when a new subtarget is added.